scripts/Deploy-MSBuild.ps1

```diff
@@ -49,6 +49,8 @@ $filesToCopyToBin = @(
     "artifacts\bin\MSBuild.Bootstrap\$configuration\$targetFramework\Microsoft.CSharp.CurrentVersion.targets"
     "artifacts\bin\MSBuild.Bootstrap\$configuration\$targetFramework\Microsoft.CSharp.targets"
     "artifacts\bin\MSBuild.Bootstrap\$configuration\$targetFramework\Microsoft.Managed.targets"
+    "artifacts\bin\MSBuild.Bootstrap\$configuration\$targetFramework\Microsoft.Managed.Before.targets"
```

Comment:
Since Before/After are both part of Microsoft.Managed.targets now, these lines are unnecessary, right?

Reply:
It's the other way around: Managed.targets now imports the other two. So they need to be deployed/updated.